version/prop.go

```diff
@@ -206,8 +204,6 @@ type Meta struct {
 	Cap int `json:"cap"`
 }
 
-var getMeta lazy.SyncValue[Meta]
-
```

Comment:
It looks like this was intended to be used by @danderson in #7254 but was forgotten; might be worth asking Dave if we should use this?

Reply:
Since Dave is on vacation, I just implemented the lazy loading in `GetMeta`